lib/config/definitions.js

```diff
@@ -186,6 +186,15 @@ const options = [
     admin: true,
     default: false,
   },
+  {
+    name: 'gitFsProtocol',
```

Comment:
What if we change the `gitFs` field to be string/enum, with possible values:

- null: no gitFs (default)
- 'http'
- 'https'
- 'ssh'

We can massage false->null and true->'https' to maintain backwards compatibility.